sensor/tests/connection/connection_test.go

```diff
@@ -125,8 +126,16 @@ func Test_ScannerDefinitionsProxy(t *testing.T) {
 		req, err := http.NewRequest(http.MethodGet, "https://localhost:8443/scanner/definitions", nil)
 		require.NoError(t, err)
 
-		resp, err := client.Do(req)
+		// Sensor may not be yet in the ready state and may reject the API call
+		err = retry.WithRetry(func() error {
```

Comment:
Not blocking but normally in test I like to use the `Eventually` functions from testify.

Reply:
Yeah, that is much better! Added that in [5052d96](https://github.com/stackrox/stackrox/pull/13532/commits/5052d9699bae870a844332dbf67eb61963085463)

Reply:
Luis, I resolve that comment and merge this PR as you are not working today and I don't want to postpone it too much. Let me know in case I missed something and I will fix it in a follow-up.